requirements.txt

```diff
@@ -0,0 +1,15 @@
+pytest
```

Comment:
The whole contents of this file is actually *only* required for installing esm with esmfold; see README instructions on `add the [esmfold] option to your pip install`. This is still nice though, but let's do this:
1. rename this file to requirements_esmfold.txt
2. Move all the stuff from setup.py `extras["esmfold"]` to this file, unpin versions as much as possible
3. Make sure it's minimal, don't think we should need something that's not curently in `extras["esmfold"]` ?
4. Read this file into list for `extras["esmfold"]`